scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -256,6 +256,11 @@ ${x}CommandBufferAppendKernelLaunchExp. The command can then be updated
 to use the new kernel handle by passing it to
 ${x}CommandBufferUpdateKernelLaunchExp.
 
+.. important::
+    When updating the kernel handle of a command you must also provide all
```

Comment:
Can we use passive voice here, and avoid using you?
Might better align with the existing documentation.